docs/source/llms/langchain/notebooks/langchain-quickstart.ipynb

```diff
@@ -262,6 +271,13 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
+      "2024/04/18 18:09:02 WARNING mlflow.langchain.utils: MLflow does not guarantee support for LLMs outside of HuggingFaceHub and OpenAI, found OpenAI\n",
```

Comment:
I remember you solved this warning right? Maybe we can remove it as the next release will exclude this warning 😄